package.json

```
@@ -10,37 +10,56 @@
     "lint:fix": "eslint . --fix",
     "start": "node src/server.js",
     "test": "jest test/**/*.test.js",
-    "test:integration": "jest test/integration"
+    "test:integration": "jest test/integration",
+    "postinstall": "patch-package",
+    "preinstall": "npx npm-force-resolutions"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mojaloop/reporting.git"
   },
   "author": "Matt Kingston <matt.kingston@modusbox.com>",
+  "contributors": [
+    "Yevhen Kyriukha <yevhen.kyriukha@modusbox.com>"
+  ],
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/mojaloop/reporting/issues"
   },
+  "resolutions": {
+    "fast-csv": "4.3.6",
+    "xmldom": "0.5.0",
+    "cheerio": "1.0.0-rc.5",
+    "parse5-htmlparser2-tree-adapter": "6.0.1"
+  },
   "homepage": "https://github.com/mojaloop/reporting#readme",
   "devDependencies": {
-    "csv-parse": "^4.10.1",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-plugin-import": "^2.21.2",
-    "jest": "^25.5.4",
-    "npm-audit-resolver": "2.2.0",
-    "supertest": "^4.0.2"
+    "csv-parse": "^4.15.3",
+    "eslint": "^7.23.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "jest": "^26.6.3",
+    "npm-audit-resolver": "2.2.1",
+    "npm-force-resolutions": "0.0.10",
+    "supertest": "^6.1.3"
   },
   "dependencies": {
     "@internal/randomphrase": "file:src/lib/randomphrase",
     "@internal/router": "file:src/lib/router",
-    "@mojaloop/sdk-standard-components": "^10.3.2",
-    "csv-stringify": "^5.5.0",
+    "@mojaloop/sdk-standard-components": "^15.6.1",
+    "chrome-page-eval": "^1.3.0",
+    "csv-stringify": "^5.6.2",
+    "ejs": "^3.1.6",
+    "html-to-xlsx": "^2.0.3",
+    "js-yaml": "^4.0.0",
     "json2xls": "^0.1.2",
-    "koa": "^2.13.0",
-    "koa-sendfile": "^2.0.1",
-    "mysql2": "^2.1.0",
-    "object.fromentries": "github:es-shims/Object.fromEntries#v2.0.2",
-    "string.prototype.matchall": "github:es-shims/String.prototype.matchAll#v4.0.2"
+    "koa": "^2.13.1",
+    "koa-sendfile": "^3.0.0",
+    "mysql2": "^2.2.5",
+    "node-table-to-csv": "^0.1.0",
+    "object.fromentries": "github:es-shims/Object.fromEntries#v2.0.4",
+    "patch-package": "^6.4.7",
+    "puppeteer": "^8.0.0",
+    "string.prototype.matchall": "github:es-shims/String.prototype.matchAll#v4.0.4"
```

Looks like this is no longer used as a result of these changes
```suggestion
    "string.prototype.matchall": "github:es-shims/String.prototype.matchAll#v4.0.4"
```

---

done in PR https://github.com/mojaloop/reporting/pull/17